mbuild/utils/io.py

```diff
@@ -133,6 +145,13 @@ def import_(module):
 except ImportError:
     has_openbabel = False
 
+try:
+    import foyer
+    has_foyer = True
+    del foyer
+except ImportError:
+    has_foyer = False
+
```

Comment:
Just curious - why are these `try` blocks not indented?

Reply:
The try-except blocks are not part of a function and are there only to initialize the `has_foyer` variable. I still need to mark the appropriate tests, but this way we can import `has_foyer` into the test scripts and skip tests if foyer is not installed.